pkg/services/authz/zanzana/common/info.go

```diff
@@ -32,3 +32,10 @@ var VerbMapping = map[string]string{
 	utils.VerbDelete:           "delete",
 	utils.VerbDeleteCollection: "delete",
 }
+
+var RelationToVerbMapping = map[string]string{
+	"read":   utils.VerbGet,
+	"create": utils.VerbCreate,
+	"write":  utils.VerbUpdate,
+	"delete": utils.VerbDelete,
```

Comment:
Nit we should use constants here, also if you could fix that in VerbMapping that would be great! https://github.com/grafana/grafana/blob/main/pkg/services/authz/zanzana/common/tuple.go#L31-L34